### PR TITLE
ci: remove EUI initialization

### PR DESCRIPTION
**Pull request by markov00**
I've found that due to a generic import we where also importing and initialising part of EUI in our VR server app.
This should reduce the load time of a page by ~200ms for each test.
I will progressively update this PR to track the improvements.

These tests are compared against an [average master branch CI time](https://kibana-ci.elastic.co/job/elastic+elastic-charts+pull-request/2124/):
- `!all.stories.ts`:
   - overall time: 17min 
    - testing time:  570s
- `all.stories.ts`:
   - overall time: 12min 
    - testing time:  278s

## [1st test](#1-test) c32b41a [build](https://kibana-ci.elastic.co/job/elastic+elastic-charts+pull-request/2125/)
- removal of EUI icons preload
- importing of preferences from a deeper level 

results:
- `!all.stories.ts`:
   - overall time: 14min 
    - testing time:  406s
- `all.stories.ts`:
   - overall time: 11min 
    - testing time:  193s

- small visual regression defects on a single test (advanced annotation markers) due to minimal pixel difference on the marker arrow

## [2nd test](#2-test) 1589574 [build](https://kibana-ci.elastic.co/job/elastic+elastic-charts+pull-request/2126/)
- extracted the CSS into a separate file (it took a decent amount if time to programmatically add the style tag from code, this extract all the style into a css file and loads it in parallel with js)
- used `es2019` as TS target (it probably reduce a bit the bundle size) and the script parsing

results:
- `!all.stories.ts`:
   - overall time: 13min 
    - testing time:  363s
- `all.stories.ts`:
   - overall time: 11min 
    - testing time:  167s
  same visual regression defects as the first test

## [3rd test](#3-test) test 2b3fc7c [build](https://kibana-ci.elastic.co/job/elastic+elastic-charts+pull-request/2127/)
results:
- `!all.stories.ts`:
   - overall time: 15min 
    - testing time:  366s
- `all.stories.ts`:
   - overall time: 11min 
    - testing time:  167s
- no visual regressions issues

## [4th test](#4-test)

- added `fibers` https://www.npmjs.com/package/fibers to speedup webpack server. 
- 
On a local environment the speed increased by 4x. Unfortunately `node-fibers` are deprecated and cannot be used on node 16, but for now, if works, it is a great improvement: https://github.com/laverdet/node-fibers

<details><summary>Local Env without fibers: General output time took 1 min, 52.36 secs</summary>

```
 SMP  ⏱  Loaders
modules with no loaders took 1 min, 20.53 secs
  module count = 2408
css-loader, and
postcss-loader, and
sass-loader took 1 min, 11.3 secs
  module count = 2
ts-loader took 1 min, 10.68 secs
  module count = 757
file-loader took 0.16 secs
  module count = 4
style-loader, and
css-loader, and
postcss-loader, and
sass-loader took 0.099 secs
  module count = 2
html-webpack-plugin took 0.037 secs
  module count = 1
```
</details>

<details><summary>Local Env without fibers: General output time took 35.26 secs</summary>

```
 SMP  ⏱

 SMP  ⏱  Loaders
modules with no loaders took 29.028 secs
  module count = 2408
ts-loader took 28.81 secs
  module count = 757
css-loader, and
postcss-loader, and
sass-loader took 23.43 secs
  module count = 2
style-loader, and
css-loader, and
postcss-loader, and
sass-loader took 0.082 secs
  module count = 2
file-loader took 0.062 secs
  module count = 4
html-webpack-plugin took 0.035 secs
  module count = 1
```
</details>

<details><summary><a href="https://kibana-ci.elastic.co/job/elastic+elastic-charts+pr-vrts/2123">CI Env without fibers</a>: General output time took 2 mins, 15.098 secs</summary>

``` 
17:03:09  SMP  ⏱  Loaders
17:03:09 modules with no loaders took 1 min, 44.54 secs
17:03:09   module count = 2410
17:03:09 mini-css-extract-plugin, and 
17:03:09 css-loader, and 
17:03:09 postcss-loader, and 
17:03:09 sass-loader took 1 min, 31.49 secs
17:03:09   module count = 2
17:03:09 css-loader, and 
17:03:09 postcss-loader, and 
17:03:09 sass-loader took 1 min, 31.057 secs
17:03:09   module count = 2
17:03:09 ts-loader took 1 min, 16.96 secs
17:03:09   module count = 754
17:03:09 file-loader took 0.088 secs
17:03:09   module count = 4
17:03:09 html-webpack-plugin took 0.032 secs
17:03:09   module count = 1
```
</details>

<details><summary><a href="https://kibana-ci.elastic.co/job/elastic+elastic-charts+pull-request/2132/">CI Env with fibers</a>: General output time took 1 min, 9.96 secs</summary>

```
16:37:08  SMP  ⏱  Loaders
16:37:08 modules with no loaders took 38.98 secs
16:37:08   module count = 2410
16:37:08 ts-loader took 36.31 secs
16:37:08   module count = 754
16:37:08 mini-css-extract-plugin, and 
16:37:08 css-loader, and 
16:37:08 postcss-loader, and 
16:37:08 sass-loader took 29.47 secs
16:37:08   module count = 2
16:37:08 css-loader, and 
16:37:08 postcss-loader, and 
16:37:08 sass-loader took 29.054 secs
16:37:08   module count = 2
16:37:08 file-loader took 0.037 secs
16:37:08   module count = 4
16:37:08 html-webpack-plugin took 0.024 secs
16:37:08   module count = 1
```
</details>

### CODE REVIEW

File: integration/server/generate/vrt_page_template.js [hunk 1]
```diff
@@ -19,22 +19,6 @@ import '../../storybook/style.scss';
...
-  arrowRight: require('@elastic/eui/es/components/icon/assets/arrow_right').icon,
-  iInCircle: require('@elastic/eui/es/components/icon/assets/iInCircle').icon,
-  tokenKey: require('@elastic/eui/es/components/icon/assets/tokens/tokenKey').icon,
-  filter: require('@elastic/eui/es/components/icon/assets/filter').icon,
-  starFilled: require('@elastic/eui/es/components/icon/assets/star_filled').icon,
-  pencil: require('@elastic/eui/es/components/icon/assets/pencil').icon,
-  visualizeApp: require('@elastic/eui/es/components/icon/assets/app_visualize').icon,
-});
-
-document.querySelector('html')?.classList.add('disable-animations');
```
Comment (nickofthyme) [inline integration/server/generate/vrt_page_template.js:37]: Why is this to be removed? This is to prevent icons causing diffs in VRT.

**Review by nickofthyme — CHANGES_REQUESTED** [review]

File: integration/server/generate/vrt_page_template.js (hunk 1 above)
Comment (markov00) [reply integration/server/generate/vrt_page_template.js:37]: to reduce the loading time on every chart
From what I'm testing only one test is getting failures on the icon. We can probably replace that icon with a simple, non EUI icon.

Comment (markov00) [reply integration/server/generate/vrt_page_template.js]: let me tray again including just this part https://github.com/elastic/elastic-charts/pull/1366/commits/2b3fc7c045befa1894a4c9db574eb1b216ab4750

The `disable-animations` class was already in the `html` tag so is safe to remove that last line

Comment (nickofthyme) [reply integration/server/generate/vrt_page_template.js]: Ok, but I don't think the error would be immediate. I would guess several will fail eventually & randomly if we do not preload the icons.

Comment (markov00) [reply integration/server/generate/vrt_page_template.js]: with the preload icons it looks anyway good: 2b3fc7c

**Review by nickofthyme — APPROVED** [review]

**Pull request merged**